Shell/main.cpp

```diff
@@ -147,6 +147,43 @@ static int sh_unset(int argc, char** argv)
     return 0;
 }
 
+static String expanded_tilde_expression_from(char* arg)
+{
+    int len = strlen(arg);
+    ASSERT(len);
+
+    StringBuilder login_name;
+    int first_slash_index = len;
+    for (int i = 1; i < len; ++i) {
+        if (arg[i] == '/') {
+            first_slash_index = i;
+            break;
+        }
+        login_name.append(arg[i]);
+    }
+
+    StringBuilder path;
+    for (int i = first_slash_index; i < len; ++i)
+        path.append(arg[i]);
+
+    if (login_name.length() == 0) {
```

Comment:
`.is_empty()`

Reply:
Added that function!

Reply:
I guess it can be useful, but I meant `StringView::is_empty()`. You should really stop using `StringBuilder`s here, except maybe once in the end to build the final result.

Reply:
What is the best way to split the string into two strings at the first occurrence of a "/" then? I can't use something like split_view("/") because that gets rid of all "/"s which isn't what we want...